app/controllers/stories_controller.rb

```diff
@@ -330,4 +331,58 @@ def assign_podcasts
   def assign_classified_listings
     @classified_listings = ClassifiedListing.where(published: true).select(:title, :category, :slug, :bumped_at)
   end
+
+  def set_json_ld
+    @user_json_ld = {
+      "@context": "http://schema.org",
+      "@type": "Person",
+      "mainEntityOfPage": {
+        "@type": "WebPage",
+        "@id": "https://dev.to/<%= user_url(@user) %>"
+      },
+      "url": "<%= user_url(@user) %>",
+      "sameAs": [],
+      "image": "<%= ProfileImage.new(@user).get(width: 320) %>",
+      "name": "<%= @user.name %>",
+      "email": "",
+      "jobTitle": "",
+      "description": "<%= @user.summary.presence || ['404 bio not found'].sample %>",
+      "disambiguatingDescription": [],
+      "worksFor": [
+        {
+          "@type": "Organization"
+        },
+      ],
+      "alumniOf": ""
+    }
+    user_profile_details_json_ld
+    user_same_as_json_ld
```

Comment:
Since these methods are actually modifying the instance of `@user_json_ld`, you might want to consider renaming the method names to suggest that. Maybe `set_user_profile_json_ld` and `set_user_same_as_json_ld`? Or something less verbose if can come up with it ;)